dev-env/dev_env_vars.yml

```diff
@@ -23,5 +23,9 @@ kive_httpd_user: kive
 kive_httpd_group: kive
 # - package variables
 slurmbuilddir: "/root"
+slurmnodes:
+    - name: head
+      memory: "7000"
```

Comment:
Please add a comment with the memory units, presumably MB.

Reply:
This just gets substituted into the config file template, so the units are the same as what Slurm takes (presumable, yes, MB).